src/features/calendar/components/CalendarDayView/Day/Event.tsx

```diff
@@ -38,75 +39,96 @@ const Event = ({ event }: { event: ZetkinEvent }) => {
   }
 
   return (
-    <Box
-      display="flex"
-      flexDirection="row"
-      justifyContent="space-between"
-      padding={2}
-      sx={{ backgroundColor: 'white', borderRadius: '3px' }}
-      width="100%"
+    <NextLink
+      href={`/organize/${event.organization.id}/${
+        event.campaign ? `projects/${event.campaign.id}` : 'standalone'
```

Comment:
This link doesn't look correct to me. When there is no campaign, the output will be incorrect. It should be `/organize/ID/projects/standalone/events/ID` but just becomes `/organize/ID/standalone/events/ID` (missing `projects`), am I right?

Reply:
You're right! fixing it!